internal/clients/team3/client.go

```diff
@@ -18,9 +18,9 @@ type client struct {
 }
```

Comment:
I suppose this is due to broken backwards compatibility?

Reply:
Yup, function name change